Web/HTML1/AccountForm.html

```diff
@@ -0,0 +1,193 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="ISO-8859-1">
+<title></title>
+</head>
+<body>
+	<table>
+		<tr>
+			<td colspan=2>Create Your Account Here</td>
+		</tr>
+		<tr>
+			<td align="right">*Email ID: </font></td>
+			<td><input type="text"></td>
+		</tr>
+		<tr>
+			<td></td>
+			<td><font color="gray" size="2">Enter your email here</font></td>
+		</tr>
+		<tr>
+			<td></td>
+			<td><font size="2"> Please specify your Email Id</font></td>
+		</tr>
+		<tr>
+			<td align="right">*Password:</td>
+			<td><input type="password"></td>
+		</tr>
+		<tr>
+			<td align="right">*Confirm Password:</td>
+			<td><input type="text"></td>
+		</tr>
+		<tr>
+			<td colspan=2>
+				<hr>
+			</td>
+		</tr>
+
+		<tr>
+			<td colspan=2>Contact Detail<font size="2"
+				color="gray"> this is where recruiters and employee will
+					contact you for job</font></td>
+		</tr>
+		<tr>
+			<td align="right">Your Name</td>
+			<td><input type="text"></td>
+		</tr>
+		<tr>
+			<td align="right">*Current Location:</td>
+			<td colspan="1"><select><option>Select</option></select><input
+				type="checkbox" size="1">Not an Indian</td>
+		</tr>
+		<tr>
+			<td align="right">Mobile</td>
+			<td><input type="text" size="25"></td>
+		</tr>
+		<tr>
+			<td></td>
+			<td><font size="2">If you don't have mobile then enter <a
+					href=""><font size="3">Landline number</font></a></font></td>
+		</tr>
+
+		<tr>
+			<td colspan=2>
+				<hr>
+			</td>
+		</tr>
+
+
+		<tr>
+			<td colspan=2>Current Professional Details</td>
```

Comment:
All Element of this section have different-2 size, width should be same throughout the section.